### ISSUE TITLE
请问代码块的宽度怎么修改？

### ISSUE THREAD

**Issue by One010101** [gilbertohasnofb/typora-panda-theme#5]
请问代码块的宽度怎么修改？有时候代码太长，折行后不好看

**Comment by gilbertohasnofb** [comment]
Hello, thanks for your message. Unfortunately I do not speak Chinese, but I can understand your question by using Google Translate. To change the width, open the file `panda.css `, and edit lines 1303 and 1313: `max-width: 800px;`

Change `800px` to a higher value such as `1000px` or `1200px`, for instance. This will change the width of the theme.

Let me know if this works.

See the similar question: https://github.com/gilbertohasnofb/typora-panda-theme/issues/1

---- 

您好，谢谢您的留言。 不幸的是，我不会说中文，但是通过使用Google翻译，我可以理解您的问题。要更改宽度，请打开文件panda.css，然后编辑第1303和1313行: `max-width: 800px;`

例如，将“ 800px”更改为更高的值，例如“ 1000px”或“ 1200px”。 这将改变主题的宽度。

让我知道这个是否奏效。

看到类似的问题： https://github.com/gilbertohasnofb/typora-panda-theme/issues/1

**Comment by One010101** [comment]
Thank you for your help, I successfully modified the code， and thank you for bringing such a beautiful theme

**Comment by One010101** [comment]
I changed the font to JetBrains Mono. （https://www.jetbrains.com/lp/mono/）
It was a bumpy process. I just renamed the downloaded JetBrains Mono to the name of your font file and replaced your font file, then changed the color of the code keywords and changed Typora's own window.css file to make the directory font smaller. I am very happy with the current theme
![2021-04-22_23-59-31](https://user-images.githubusercontent.com/44254059/115746833-3ea8c780-a3c7-11eb-814c-1f987e557239.png)
